tez-mapreduce/src/main/java/org/apache/tez/mapreduce/hadoop/MRJobConfig.java

```diff
@@ -131,6 +131,11 @@ public interface MRJobConfig {
 
   public static final String CACHE_ARCHIVES_VISIBILITIES = "mapreduce.job.cache.archives.visibilities";
 
+  /**
+   * Used by Hadoop's MagicS3Guard and Staging committers to set a job-wide UUID
+   */
+  public static final String FS_S3A_COMMITTER_UUID = "fs.s3a.committer.uuid";
```

Comment:
I don't think this option should be "S3" specific, even if this is implemented due to S3
the new option's name should tell this UUID is job-scoped